site/profile/manifests/efa.pp

```diff
@@ -0,0 +1,41 @@
+class profile::efa {
+  package { 'libibverbs-utils':
+    ensure => 'installed',
+  }
+
+  package { 'rdma-core-devel':
+    ensure => 'installed',
+  }
+
+  package { 'librdmacm-utils':
+    ensure => 'installed',
+  }
+
+  exec { 'download-efa-driver':
+    command => 'curl -O https://efa-installer.amazonaws.com/aws-efa-installer-1.30.0.tar.gz && tar -xf aws-efa-installer-1.30.0.tar.gz && cd aws-efa-installer',
+    cwd     => '/tmp',
+    creates => '/tmp/aws-efa-installer',
+    path    => ['/bin', '/usr/bin', '/sbin','/usr/sbin'],
+  }
+
+  exec { 'install-efa-driver':
```

Comment:
The EFA driver installation is currently being done in the cloud-init because we thought it would require a reboot for the kernel to load the modules. Are the module loaded by the EFA installer? Should we make sure they are loaded like we do for GPU drivers - https://github.com/ComputeCanada/puppet-magic_castle/blob/main/site/profile/manifests/gpu.pp#L57